generators/profilegen/src/resources/common/configurable.rb

```diff
@@ -50,7 +50,16 @@ def reset!(options = {})
         instance_variable_set(:"@#{key}", options.fetch(key, default_value))
       end
 
-      default_value = get_credentials(self.tenant_id, self.client_id, self.client_secret, self.active_directory_settings)
+      fail ArgumentError, 'tenant_id is nil' if self.tenant_id.nil?
```

Comment:
can we give an indication on how to fix it? set the env vars or pass it in options?

Reply:
also maybe match the message from https://github.com/Azure/azure-sdk-for-ruby/blob/ab9cdb6ef0f53741d76caab9c9e598f9781ca320/runtime/ms_rest_azure/lib/ms_rest_azure/credentials/application_token_provider.rb#L50